activiti-cloud-modeling-service/activiti-cloud-services-modeling/activiti-cloud-services-modeling-rest/src/test/java/org/activiti/cloud/services/modeling/rest/controller/ConnectorValidationControllerIT.java

```diff
@@ -65,11 +67,14 @@ public class ConnectorValidationControllerIT {
     @MockBean
     private SecurityManager securityManager;
 
+    private MockMvc mockMvc;
+
     @BeforeEach
     public void setUp() {
         webAppContextSetup(context);
```

Comment:
is this still needed after the introduction of `mockMvc = MockMvcBuilders.webAppContextSetup(context).build();`?